pkg/sql/plan/function/builtin/unary/load_file_test.go

```diff
@@ -109,7 +109,7 @@ func TestLoadFile(t *testing.T) {
 	}
 
 	// Test Error
-	size := 1024 * 1024 * 1
+	size := 1<<(10*3) + 1
```

Comment:
Why do you bother define those consts if you are not using them?  :)

Reply:
> Try a 1GB file.

ok